packages/illustrations/package.json

```diff
@@ -47,7 +47,6 @@
         "case-anything": "^2.1.10",
         "jest": "^27.4.5",
         "npm-run-all": "^4.1.5",
-        "prettier": "^2.2.1",
```

Comment:
This one also is used by the illustration import scripts. Although it will still resolve from the root, it's good to list it, as we depend on that version when importing illustrations.